app/models/loan.rb

```diff
@@ -237,8 +237,12 @@ def thumb_path
     else "/assets/ww-avatar-watermark.png" end
   end
 
+  def ensure_currency
+    currency || Currency.find_by(code: 'USD')
+  end
```

Comment:
This feels a bit hackish. Can we instead handle this through a migration? `currency` seems like it should be null: false in fact. How can a loan not have a currency?